internal/service/networkpeering/resource_network_peering_test.go

```diff
@@ -244,41 +246,41 @@ func testAccCheckMongoDBAtlasNetworkPeeringExists(resourceName string, peer *mat
 	}
 }
 
-func testAccMongoDBAtlasNetworkPeeringConfigAWS(orgID, projectName, providerName, vpcID, awsAccountID, vpcCIDRBlock, awsRegion string) string {
+func configAWS(orgID, projectName, providerName, vpcID, awsAccountID, vpcCIDRBlock, awsRegion string) string {
 	return fmt.Sprintf(`
-		resource "mongodbatlas_project" "my_project" {
-			name   = %[2]q
-			org_id = %[1]q
-		}
-		resource "mongodbatlas_network_container" "test" {
-			project_id   		  = mongodbatlas_project.my_project.id
-			atlas_cidr_block  = "192.168.208.0/21"
-			provider_name		  = "%[3]s"
-			region_name			  = "%[7]s"
-		}
+	resource "mongodbatlas_project" "my_project" {
+		name   = %[2]q
+		org_id = %[1]q
+	}
+	resource "mongodbatlas_network_container" "test" {
+		project_id   		  = mongodbatlas_project.my_project.id
+		atlas_cidr_block  = "192.168.208.0/21"
```

Comment:
This value is hardcoded and don't match the value I find in the UI by going to:
Project -> NetworkAccess -> Peering -> AddPeeringConnection
image.png

There I find:
dev: 192.168.240.0/21
prod: 192.168.248.0/21

Reply:
I think in the UI it shows a default/available CIDR block, but you can use a custom one if it adheres to the constraints.